src/main/java/uk/gov/hmcts/reform/fees/client/FeesClient.java

```diff
@@ -27,22 +29,45 @@ public FeesClient(
         FeatureToggleService featureToggleService,
         @Value("${fees.api.service:}") String service,
         @Value("${fees.api.jurisdiction1:}") String jurisdiction1,
-        @Value("${fees.api.jurisdiction2:}") String jurisdiction2
+        @Value("${fees.api.jurisdiction2:}") String jurisdiction2,
+        @Value("${fees.api.jurisdiction-fast-track-claim:}") String jurisdictionFastTrackClaim
     ) {
         this.feesApi = feesApi;
         this.service = service;
         this.jurisdiction1 = jurisdiction1;
         this.jurisdiction2 = jurisdiction2;
+        this.jurisdictionFastTrackClaim = jurisdictionFastTrackClaim;
         this.featureToggleService = featureToggleService;
     }
 
     public FeeLookupResponseDto lookupFee(String channel, String event, BigDecimal amount) {
-        String keyword = event.equalsIgnoreCase("issue")
-            ? "MoneyClaim"
-            : "HearingSmallClaims";
-
         if (featureToggleService.isFeatureEnabled("fee-keywords-enable")) {
-            return this.feesApi.lookupFee(service, jurisdiction1, jurisdiction2, channel, event, keyword, amount);
+            String keyword;
+            String jurisdiction2;
+
+            if (featureToggleService.isLipVLipEnabled() && "hearing".equalsIgnoreCase(event) && AllocatedTrack.FAST_CLAIM == AllocatedTrack.getAllocatedTrack(
```

Comment:
perhaps instead of having these as hardcode strings here we could put them in constants? i mean "hearing, "FastTrackHrg", "issue",   "MoneyClaim", and "HearingSmallClaims". or you can go even further and create Event enum and hearingKey enum. then we can use those in test too so that in the instance when the key is changed we change it only in one place for example